src/oauth.rs

```diff
@@ -0,0 +1,60 @@
+//use crate::token;
+use crate::settings;
+
+use oauth2::{
+    AsyncCodeTokenRequest, AuthUrl, AuthorizationCode, ClientId, ClientSecret, CsrfToken,
+    PkceCodeChallenge, RedirectUrl, Scope, TokenResponse, TokenUrl,
+};
+
+use oauth2::basic::BasicClient;
+//use oauth2::reqwest::async_http_client;
+//use url::Url;
+use url;
+// Create an OAuth2 client by specifying the client ID, client secret, authorization URL and
+// token URL.
+fn fun() -> Result<(), url::ParseError>{
```

Comment:
We're not creating an OAuth2 client. We're creating a server. We should be:

* accepting a posted token
* verifying the signature and scope
* composing the email and claims result

And if we hit an error, we should be passing the token off for external validation via the (configuration option specified) FxA default Verifier, where we would return the result back to the user.

These functions and tasks are outlined in https://github.com/mozilla-services/tokenserver/blob/master/tokenserver/verifiers.py#L175 

It's a bit confusing because `Client` in this case means our internal client that is calling the remote FxA Verifier service.